JoinColony/colonyCDapp
# Fix: update account avatar upload flow

## Description

This PR will conditionally show the avatar upload functionality in the User Account page.

![new](https://github.com/user-attachments/assets/21be1e0c-707f-45a1-96a3-30187bf51a39)

## Testing

1. Visit the User Account page http://localhost:9091/account/profile
2. You should see the following buttons by default:
- Change avatar
- Remove avatar (if an avatar has been uploaded)
![image](https://github.com/user-attachments/assets/a5fb42e6-e0f6-4b56-b40a-f72c22dddb41)
3. Clicking "Change avatar" should reveal our file upload component
4. Clicking "Remove avatar" should remove the avatar 

Resolves #2872
